ge/templates/ge/add_fund.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'ge/base.html' %}
+
+{% block content %}
+
+{% if messages %}
+<div class="box">
```

Comment:
@ztucker4 Please add some CSS to go along with this class - the message(s) need something to better offset them from the form.  Maybe something like what we used in the OH staff project:
```
.box {
    margin: 10px;
    padding: 10px;
    border: 1px solid black;
}
```